modules/system/console/PluginInstall.php

```diff
@@ -52,7 +52,10 @@ public function handle()
          * Migrate plugin
          */
         $this->output->writeln(sprintf('<info>Migrating plugin...</info>', $code));
-        PluginManager::instance()->loadPlugins();
+        $pluginManager = PluginManager::instance();
+        $pluginManager->loadPlugins();
+        $plugin = $pluginManager->findByIdentifier($code);
+        $pluginManager->registerPlugin($plugin, $code);
```

Comment:
Can you move this above the migrate plugin step and add its own comment "Ensure plugin is registered"?

Reply:
done!